ibm/data_source_ibm_container_vpc_cluster_test.go

```diff
@@ -10,16 +10,18 @@ import (
 
 func TestAccIBMContainerVPCClusterDataSource_basic(t *testing.T) {
 	clusterName := fmt.Sprintf("terraform_%d", acctest.RandIntRange(10, 100))
-	vpcID := fmt.Sprintf("vpc_%d", acctest.RandIntRange(10, 100))
-	flavor := fmt.Sprintf("c.%d", acctest.RandIntRange(10, 100))
-	zoneID := fmt.Sprintf("zone.%d", acctest.RandIntRange(10, 100))
-	subnetID := fmt.Sprintf("subnet.%d", acctest.RandIntRange(10, 100))
+	randint := acctest.RandIntRange(10, 100)
+	vpc := fmt.Sprintf("vpc-%d", randint)
```

Comment:
Add the prefix terraform to vpc, subnets...where ever u create a dynamic name to indicate its created using Terraform testcases